### ISSUE TITLE
about results(Average AUCs in % with StdDevs (over 10 seeds) )

### ISSUE THREAD

**Issue by tommying** [lukasruff/Deep-SVDD-PyTorch#17]
Hi, I am mystified by this (Average AUCs in % with StdDevs (over 10 seeds) ).

Do you mean to get the average results over 10 seeds. 
If then, what should be the range of seeds?  (1 - 10 ?  or 10 -20 ?). The results varied widely, because of different seed settings.
Can you help me with this question?  Thank you.

**Comment by dimimal** [comment]
I have the same question as well! If someone from the team could provide any help, that would be great!

**Comment by omid-ghozatlou** [comment]
This is also my question and I think its related to collect_results.py in the utils folder. and I have another question: If we set seed to any number except -1, the result must not change by rerunning. however when I use preparing the results of both pretraining and training change widely. it worth mentioning that if you set pretraining False, the network is trained without any pre-trained weights and the result of many times training are the same for the specific seed value. (of course except -1)
Anybody knows why seed does not work for autoencoder?

**Comment by omid-ghozatlou** [comment]
According to collect_results.py range of seed is 1-10

**Comment by Longchao1047027852** [comment]
> 
I also have such problems. Should the author publish the seeds, otherwise the training results may deviate due to different seeds.
